src/resmom/mom_main.c

```diff
@@ -10826,9 +10833,13 @@ mom_topology(void)
 		}
 		free(lbuf);
 	}
+bad:
+#ifndef WIN32
 	hwloc_free_xmlbuffer(topology, xmlbuf);
 	hwloc_topology_destroy(topology);
+#else
+	;
```

Comment:
Why add a "#else" that does nothing?

Reply:
The goto label is expecting a statement. On Windows, we need the label but nothing needs to be addressed. Hence the empty else block.

Reply:
Curious. OK, thanks.